docs/pages/index.mdx

```diff
@@ -31,15 +37,27 @@ import { HomePage } from "vocs/components";
 
 # Quickstart
 
-## 1. Clone the frames.js starter template (with local debugger)
+## Bootstrap the project from template using the CLI tool
 
-Run to clone the starter into a new folder called `framesjs-starter`
+Run one of the commands below based on your preferred package manager and then follow the steps in the terminal.
 
-```bash
-npx degit github:framesjs/frames.js/examples/framesjs-starter#main framesjs-starter
+### Using npm
+
+```sh
+npm init frames
+```
+
+### Using yarn
+
+```sh
+yarn create frames
 ```
 
-or [clone from github](https://github.com/framesjs/frames.js/tree/main/examples/framesjs-starter)
+### Using pnpm
+
+```sh
+pnpm create frames
+```
```

Comment:
you can create a block of instructions like this 

<img width="519" alt="image" src="https://github.com/framesjs/frames.js/assets/5469870/f60df093-fb80-4e00-bb84-ab5cd6dab73c">

<img width="757" alt="image" src="https://github.com/framesjs/frames.js/assets/5469870/0150fc00-223c-44a1-b9b9-d94dacff4d6a">

Reply:
Done.